extensions/resteasy-classic/resteasy/deployment/src/main/java/io/quarkus/resteasy/deployment/ResteasyBuiltinsProcessor.java

```diff
@@ -109,6 +109,7 @@ void vertxProviders(BuildProducer<ResteasyJaxrsProviderBuildItem> providers, Cap
     @BuildStep(onlyIf = IsDevelopment.class)
     void setupExceptionMapper(BuildProducer<ResteasyJaxrsProviderBuildItem> providers, HttpRootPathBuildItem httpRoot,
             ExceptionMapperRecorder recorder) {
+        System.out.println("---- PRODUCING");
```

Comment:
I'm pretty sure we don't want that and the other debug changes :).

Reply:
damn!!! *slaps myself in the face*